Bio/KDTree/KDTree.py

```diff
@@ -98,10 +98,13 @@ def get_radii(self):
         Return the list of distances from center after
         a neighbor search.
         """
-        a = self.kdt.get_radii()
-        if a is None:
+        n = self.kdt.get_count()
+        if n == 0:
```

Comment:
How about ``if not n:`` which I understand is consider more Pythonic?

Reply:
Either way is fine I guess. Since n is an integer number, I thought it makes more sense to compare to 0 than to None.

Reply:
As you prefer. This module is a Python / C mixture, so a more C like style is understandable.